src/app/components/articles/article-type1/article-type1.component.html

```diff
@@ -0,0 +1,11 @@
+
+<div class="article-type1">
+    <img src="https://media.istockphoto.com/id/516358950/fr/photo/partie-de-sucre-blanc.jpg?s=612x612&w=0&k=20&c=PQdHZX3hIDHlQQHHoHJNAGxyQun5ORH_4kXJteRYyUU=">
+    <div class="blur">
+        <div class="description">
+            <h2>Sweet !</h2>
+            <h4>Le sucre c'est trop bien</h4>
```

Comment:
using h means that you have a h1 a h3 and a h5 in your pages. I think it would be probably better to ordonate those h hierarchy

Reply:
For the titles, do I have to change to h1,h2 and change the font size, or use p tag and set a different font size for each paragraph ?